pt/README.md

```diff
@@ -8,15 +8,15 @@
 
 ## Translation
 
-This tutorial has been translated from English to Portuguese by a group of awesome volunteers. 
+Este tutorial foi traduzido do Inglês para o Português por um grupo incrível de voluntários. 
 
-Special thanks for help goes out to:
+Um agradecimento especial vai para Special thanks for help goes out to:
```

Comment:
I think you forgot to take off the English part: "Special thanks for help goes out to:".

Reply:
Sure! Thank you :*